### ISSUE TITLE
Elements should have sufficient color contrast (followup)

### ISSUE THREAD

**Issue by akslay** [az-digital/arizona-bootstrap#321]
## Problem/Motivation
Related to #309

The horizontal rule element stays the same color regardless of background color, making it inaccessible on darker background colors.

## Proposed resolution
Add the `<hr>` tag to the SCSS that ensures proper color contrast; it should match the color of the text.

In general, might need to just throw every possible element on a dark background color to make sure we caught everything (I'm coming across these as I add elements to the CKEditor in AZ Quickstart 2.0).

**Comment by trackleft** [comment]
Is there an WCAG recommendation for this?

**Comment by akslay** [comment]
I didn't find anything about horizontal rules specifically on WCAG, but I would think an `<hr />` should have the same level of color contrast as text, right?

**Comment by akslay** [comment]
Another item that needs to leverage the built-in color contrast: text inside tables.
![Screen Shot 2022-05-25 at 2 58 33 PM](https://user-images.githubusercontent.com/52931751/170374222-03a5ec65-d361-4125-a32a-1c1baae2619a.png)
